public/javascripts/items.js

```diff
@@ -50,7 +50,7 @@ const descriptionEdited = event => {
 
 const completedClicked = event => {
   const element = $( event.target )
-  const id = element.data( 'id' )
+  const id = element.parent().data().id
```

Comment:
Also, why doesn't the element just have the id for itself, given that it's the target of this event? I just don't like hanging our code on the notion that the html structure here won't change.